app.getspace.Space.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is something that ideally should be part of your tarball, but it's not a blocker.